### ISSUE TITLE
Set PATH on a per solution/project basis

### ISSUE THREAD

**Issue by dmcminn** [csoltenborn/GoogleTestAdapter#201]
Hello,

Apologies if this is the wrong place, I couldn't find a discussion forum or place to just ask questions.

Is it possible to have the "PATH extension" option set on a per solution or project basis? I'm using it to pull in the path to an externally built DLL rather than copying the DLLs into my target directory. Using the PATH extension works fine to allow the DLLs to be found during test. But depending on the project or which release we may be working with an older version of the libraries so having one fixed PATH extension is not feasible. Also I would not want that path added to every test as some projects do not use the external library (although in that case it doesn't matter technically, it's just for "cleanliness").

Thanks.

**Comment by csoltenborn** [comment]
This is already possible - see [Configuration section](https://github.com/csoltenborn/GoogleTestAdapter#configuration) of the docs...

Let me know if this doesn't suffice to fulfill your use case.

**Comment by dmcminn** [comment]
Thank you, I missed that. I was able to get what I wanted working as a solution setting file. The only thing which I miss is the ability to use environment variables in the "PathExtension" setting. I tried both Visual Studio style $(EnvVar) and Windows %EnvVar% but ended up having to hard code the path. It would be nice to have environment variable support so that the test settings can stay as they are (and go into source control) and the environment variable on each developer computer has the correct path to the DLLs.
